static/docs/command-reference/import.md

```diff
@@ -121,14 +121,23 @@ outs:
 ```
 
 Several of the values above are pulled from the original stage file
-`model.pkl.dvc` in the external DVC repository. `url` and `rev_lock` fields are
-used to specify the origin and version of the dependency.
+`model.pkl.dvc` in the external DVC repository. `url` and `rev_lock` subfields
+under `repo` are used to save the origin and version of the dependency.
 
 ## Example: fixed revisions & re-importing
 
+```dvc
```

Comment:
I would add an enter phrase above: 

"To import a specific revision of an artifact you may use:"

Reply:
Agree. Added.